diff --git a/core/binary.mk b/core/binary.mk
index 589c462..0c16be8 100644
--- a/core/binary.mk
+++ b/core/binary.mk
@@ -1482,7 +1482,7 @@ endif
 
 my_outside_includes := $(filter-out $(OUT_DIR)/%,$(filter /%,$(my_c_includes)))
 ifneq ($(my_outside_includes),)
-$(error $(LOCAL_MODULE_MAKEFILE): $(LOCAL_MODULE): C_INCLUDES must be under the source or output directories: $(my_outside_includes))
+$(warning $(LOCAL_MODULE_MAKEFILE): $(LOCAL_MODULE): Local C_INCLUDES is depreceated and must be under the source or output directories: $(my_outside_includes))
 endif
 
 # all_objects includes gen_o_objects which were part of LOCAL_GENERATED_SOURCES;
