commit a230032407edaa9483563580b7acd338d3cfe5ae
Author: vishalk17 <vishalkapadi95@gmail.com>
Date:   Mon Oct 2 07:25:55 2017 +0000
Subject: add atomic.c
Change-Id: I5a9adedf66fabb088aa05012383e10af6a6a7fda

diff --git a/libutils/atomic.c b/libutils/atomic.c
new file mode 100644
index 000000000..d34aa0081
--- /dev/null
+++ b/libutils/atomic.c
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2007 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Generate non-inlined versions of android_atomic functions.
+ * Nobody should be using these, but some binary blobs currently (late 2014)
+ * are.
+ * If you read this in 2015 or later, please try to delete this file.
+ */
+
+#define ANDROID_ATOMIC_INLINE
+
+#include <cutils/atomic.h>
